src/app/static/src/app/root.ts

```diff
@@ -149,9 +149,10 @@ export const emptyState = (): RootState => {
         downloadResults: initialDownloadResultsState()
     }
 };
+const existingState = localStorageManager.getState();
 
 export const storeOptions: StoreOptions<RootState> = {
-    state: emptyState(),
+    state: {...emptyState(), ...existingState && {language: existingState.language}},
```

Comment:
Do we have a test for this? Would be good to check that if language setting has been serialised to localStorage then it is reflected in initial state of the store.

I think this is a different test to the new one in `localStorageManager.test.ts` - it checks deserialisation but not that the the store is correctly populated.

Reply:
https://github.com/mrc-ide/hint/blob/97faa5b7a0ddec3c306219529b535c0e27ff0ba0/src/app/static/src/tests/components/header/languageMenu.test.ts#L23

Reply:
If I revert your change to L154 then `npm test` still passes - including that one in `languageMenuTest.test.ts` - which implies that at least one test is missing

Reply:
I think this case can only be tested in browser i.e selenium.

Reply:
I assume you mean because `storeOptions` isn't a function (unlike `emptyState`), so that it's already initialised by the time the tests are run.

Would be good to get @EmmaLRussell's opinion on this. If it was a no-arg function then test would be as simple as:
```typescript
       jest.spyOn(localStorageManager, "getState").mockImplementation(() => mockRootState({language: Language.pt}));
       expect((storeOptions().state as RootState).language).toBe("pt");
```

Reply:
That's a good point, it could be. It also applies to all the other modules where this pattern is used - I don't think we have tests for any of those. We'd want to keep `existingState` as an object of course, initialised when the app loads.